force-app/main/default/lwc/givingSummaryDetailPage/givingSummaryDetailPage.js

```diff
@@ -0,0 +1,6 @@
+import { api, LightningElement } from 'lwc';
+
+export default class GivingSummaryDetailPage extends LightningElement {
```

Comment:
Will this component be a stand alone page or a component on the Contact Detail Record page? If it's a component and not a stand alone page, remove the `Page` in the LWC name to make it less confusing.

Reply:
This component is for be used in ContactDetailPage, that's why the Page suffix. Im gonna make it clearer changing the name for GivingSummaryForDetailPage